### DISCUSSION TITLE
Using Personal access Tokens on multiple systems

### DISCUSSION THREAD

**Discussion by perdrix52** [community/community#23576] [New to GitHub]
<p>If I switch to using these, how can I transfer the token to other computers that I use the same github account from.</p>
<p>ATM I cannot see a way to achieve this …  I can’t use them unless I can transfer the toke between systems (Windows and Linux).</p>
<p>David</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You have two options:</p>
<ol>
<li>Use copy the token from one system to the other.</li>
<li>Create a separate token for each system.</li>
</ol>
<p>It’s really up to you what you prefer. I use option 1: I store the token in my <a href="https://keepassxc.org/">password manager of choice</a>, and when I synchronize its database between my systems the token comes along. :slightly_smiling_face:</p>
<p>If you want the option to revoke the token for one system without having to update the other option 2 is better.</p>

**Comment by perdrix52** [comment]
<p>So is it just a “long password” that can be stored in e.g. Firefox password mgr? Or if it is stored in Windows credential mgr, how to transport to Linux?</p>
<p>So for clarification can I multiple active PATs for the same Github user?</p>
<p>PS when does password access go away?</p>
<p>David</p>
<p><img src="https://user-images.githubusercontent.com/20211762/181077992-a17a4a05-ae59-45cd-8378-7becf6351b07.jpeg" alt="image001.jpg" data-base62-sha1="yffejTEm3HfJr1bCee6ZiL0F062" width="56" height="56"></p>
<p><img src="https://user-images.githubusercontent.com/20211762/181077994-62a15109-2d5b-47dd-821f-1990ea170d0c.jpeg" alt="image002.jpg" data-base62-sha1="awbpBs6dq2iCPz4tC61BtCtABRW" width="25" height="25"></p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="perdrix52" data-post="3" data-topic="198446">
<div class="title">
<div class="quote-controls"></div>
 perdrix52:</div>
<blockquote>
<p>So is it just a “long password” that can be stored in e.g. Firefox password mgr?</p>
</blockquote>
</aside>
<p>Pretty much, yes.</p>
<aside class="quote no-group" data-username="perdrix52" data-post="3" data-topic="198446">
<div class="title">
<div class="quote-controls"></div>
 perdrix52:</div>
<blockquote>
<p>So for clarification can I multiple active PATs for the same Github user?</p>
</blockquote>
</aside>
<p>Yes. And if you use them for separate purposes you should, to minimize access rights for each application.</p>
<aside class="quote no-group" data-username="perdrix52" data-post="3" data-topic="198446">
<div class="title">
<div class="quote-controls"></div>
 perdrix52:</div>
<blockquote>
<p>PS when does password access go away?</p>
</blockquote>
</aside>
<p>It already has: <a href="https://github.blog/changelog/2021-08-12-git-password-authentication-is-shutting-down/" class="inline-onebox">Git password authentication is shutting down | GitHub Changelog</a> You still need your password for the website.</p>

**Comment by jsoref** [comment]
<p>Fwiw, while a PAT appears as a “long password”, it’s a token that’s a proxy for some portion of your credentials and this allows you to add restrictions regarding how it can be used.</p>
<p>At a basic restriction, it shouldn’t be possible to log into the web site <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> with this password.</p>
<p>But, you could also restrict it to only perform some even more limited set of tasks.</p>
<p>You could create a PAT that can’t access your private github repositories – it wouldn’t be terribly useful for <code>git</code>, but you could use it with <a href="https://github.com/cli/cli" rel="noopener nofollow ugc">gh</a>.</p>
